### PR TITLE
chore: Update supporting files

### PR DESCRIPTION
**Pull request by jai**
## Pull Request Submission Checklist

Please confirm that you have done the following before requesting reviews:

- [x] I have confirmed that the PR type is appropriate for the change I am making according to the [Honest Pull Request and Commit Message Naming Conventions](https://www.notion.so/honestbank/Pull-Request-and-Commit-Message-Naming-Conventions-bd97f2cbb34c4c73b1ff3a3e384b850c).
- [x] I have typed an adequate description that explains **why** I am making this change.
- [x] I have installed and run standard pre-commit hooks that lints and validates my code.

### Description

* Update various supporting files (split from #27 to reduce its size) - doesn't seem to have helped though because the same diffs can still be seen in #27 😆 

- - -
This change is [Reviewable](https://reviewable.io/reviews/honestbank/terraform-gcp-bigquery/30)

### CODE REVIEW

**Comment by jai** [comment]
Current dependencies on/for this PR:
* main
  * **PR #29** <a href="https://app.graphite.dev/github/pr/honestbank/terraform-gcp-bigquery/29" target="_blank">Graphite</a> 
    * **PR #30** <a href="https://app.graphite.dev/github/pr/honestbank/terraform-gcp-bigquery/30" target="_blank">Graphite</a>  👈
    * **PR #27** <a href="https://app.graphite.dev/github/pr/honestbank/terraform-gcp-bigquery/27" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/honestbank/terraform-gcp-bigquery/30?utm_source=stack-comment).

**Review by thadchais — APPROVED** [review]

**Review by HeshamMeneisi — APPROVED** [review]
LGTM

File: Makefile [hunk 1]
```diff
@@ -7,7 +7,16 @@ validate:
 	terraform fmt --recursive
 
 docs:
+	rm -rf modules/*/.terraform modules/*/.terraform.lock.hcl
+	rm -rf examples/*/.terraform examples/*/.terraform.lock.hcl
 	terraform-docs -c .terraform-docs.yml .
+	cd examples/gcp_bigquery_big_lake_table/; terraform-docs markdown . --output-file README.md --output-mode inject
```
Comment (sunilhonest) [inline Makefile:13]: Can we have a .terraform-docs.yaml inside each of the examples/**/ folder and use recursive functionality of terraform-docs?

https://terraform-docs.io/user-guide/configuration/recursive/

**Review by sunilhonest — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (jai) [reply Makefile:13]: We're already using `recursive` but it seems to only support one folder (in this case it's set to `modules`) - or do you mean something else?

```
recursive:
  enabled: true
  path: modules
output:
  file: README.md
  mode: inject
formatter: "markdown"
```

**Review by sunilhonest — APPROVED** [review]

**Pull request merged**